packages/trail-fastify-graphql-plugin/lib/index.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const gql = require('fastify-gql')
+const fp = require('fastify-plugin')
+const { makeExecutableSchema } = require('graphql-tools')
+
+const { typeDefs, makeResolvers } = require('./graphql')
+
+async function graphql (server, options) {
+  const {
+    // Allow some fastify-gql options to be specified in this plugin's config.
+    graphiql,
+    path,
+    prefix,
+    // Allow a trails manager instance to be provided.
+    trailsManager
+  } = options
+  // Compile the schema.
+  const resolvers = makeResolvers({ trailsManager })
+  const schema = makeExecutableSchema({ typeDefs, resolvers })
+  // Register the gql plugin.
+  server.register(gql, { schema, graphiql, path, prefix })
```

Comment:
Can you add a couple of tests for this `prefix` option? I think you might end up with double prefixes if it's set. It should be automatically applied to all descending plugins.